benchmarks/viscoelastic_sheared_torsion/viscoelastic_sheared_torsion.prm

```diff
@@ -0,0 +1,169 @@
+# This parameter file constructs a 3D cube which is rotated around its vertical axis and sheared in the
+# xz-plane. It is adapted from Farrington et al. 2014 (Geophysics, Geochemsitry, Geosystems), section "3.4.
```

Comment:
```suggestion
# xz-plane. It is adapted from Farrington et al. 2014 (Geophysics, Geochemistry, Geosystems), section "3.4.
```

Reply:
oh, this I copied from the simple shear benchmark. So I should probably fix the typo in there too. Should I make a separate PR for that?